packages/rpc-core/src/bundle.ts

```diff
@@ -487,15 +490,15 @@ export class RpcCore {
       : u8aToU8a(value);
     const codec = this._newType(registry, blockHash, key, input, isEmpty, entryIndex);
 
-    // store the retrieved result - the only issue with this cache is that there is no
-    // clearing of it, so very long running processes (not just a couple of hours, longer)
-    // will increase memory beyond what is allowed.
-    this.#storageCache.set(hexKey, codec);
-    this.#storageCacheSize++;
```

Comment:
Since cache size is being removed here it makes the `stats` call irelevant. this should be added in `-setToCache`

Reply:
I updated the stats returned using the LRU cache size